pulpcore/cli/common/generic.py

```diff
@@ -805,6 +805,18 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: t.Optional
     expose_value=False,
 )
 
+repositories_option = resource_option(
+    "--repository",
+    "repositories",
+    context_table=PulpRepositoryContext.TYPE_REGISTRY,
+    multiple=True,
+    href_pattern=PulpRepositoryContext.HREF_PATTERN,
+    help=_(
+        "Repository to export from in the form '[[<plugin>:]<resource_type>:]<name>' or by href."
+        " Can be called multiple times."
```

Comment:
This help text is not valid. Maybe it's not worth making the option generic.

Reply:
@mdellweg thanks, I fixed the text, I just copy-pasted the original and then forgot to change it. I really think it is a generic thing, as its purpose is to be able to obtain a list of repositories in the most practical way, I see nothing too specialized about it.